JWT secret rotation (#314)
arangodb/docs: 3.7/http/general.md

curl -v -H "Authorization: bearer $(jwtgen -s <my-secret> -e 3600 -a "HS256" -c 'iss=arangodb' -c 'server_id=myclient')" http://<database-ip>:8529/_api/version
```

#### Hot-Reload of JWT Secrets

To perform a hot reload the JWT secrets of a local arangod process you may use the following 
HTTP restful API. Perform a **POST** request to reload the secret, a **GET** request may 

> be used to load information obout the currently used secrets.

Reviewer: typo: obout -> about

curl -v -H "Authorization: bearer $(jwtgen -s <my-secret> -e 3600 -a "HS256" -c 'iss=arangodb' -c 'server_id=myclient')" http://<database-ip>:8529/_api/version
```

#### Hot-Reload of JWT Secrets

To reload the JWT secrets of a local arangod process without a restart, you may use the following 
restful API. A **POST** request reloads the secret, a **GET** request may 
be used to load information about the currently used secrets:

Hot-Reload the secrets from disk:

`POST /_admin/server/jwt`

To fetch information about the currently loaded secrets:

`GET /_admin/server/jwt`

A successful response contains a JSON with the "result" field and an "error" field containing _false_.

> The field _result_ which in turn has the fields _active_ and _passive_, which contain the sha256 hashes of the loaded secrets.

Reviewer 2: ```suggestion
The field _result_ contains the sub-fields _active_ and _passive_, which contain the sha256 hashes of the loaded secrets.
```

> The field _passive_ is always an array.

Reviewer 2: ```suggestion
The _field_ active is always an object, the field _passive_ is always an array of objects. However, _passive_ may be an empty array if there is only an active secret but no passive ones.
```
